lib/matplotlib/legend_handler.py

```diff
@@ -378,12 +375,11 @@ def _create_line(self, legend, orig_handle,
     def create_artists(self, legend, orig_handle,
                        xdescent, ydescent, width, height, fontsize, trans):
         if orig_handle.get_fill() or (orig_handle.get_hatch() is not None):
-            p = self._create_patch(legend, orig_handle,
-                                   xdescent, ydescent, width, height, fontsize)
+            p = HandlerStepPatch._create_patch(orig_handle, xdescent,
+                                               ydescent, width, height)
             self.update_prop(p, orig_handle, legend)
         else:
-            p = self._create_line(legend, orig_handle,
-                                  xdescent, ydescent, width, height, fontsize)
+            p = HandlerStepPatch._create_line(orig_handle, width, height)
```

Comment:
Note that these can remain as `self._create*` even though they're static methods.

Reply:
I reverted those changes (I had forgotten that it works like that...).